upload_to_s3.py

```diff
@@ -26,7 +26,7 @@
 # don't care about but also because any datetime fields would require extra handling in
 # order to JSON-serialize them
 QUERIES = {
-    "task_orders": "select atd_tk.TASK_ORDER_DEPT, atd_tk.TASK_ORDER_ID, atd_tk.TASK_ORDER_DESC, atd_tk.TASK_ORDER_STATUS, atd_tk.TASK_ORDER_TYPE, atd_tk.TK_CURR_AMOUNT, atd_tk.CHARGED_AMOUNT, atd_tk.TASK_ORDER_BAL, buyer_tk.BYR_FDU FROM DEPT_2400_TK_VW atd_tk LEFT JOIN( SELECT DISTINCT TASK_ORD_CD, BYR_FDU FROM REL_BUYER_SELLER_FDU_TK) buyer_tk ON atd_tk.TASK_ORDER_ID = buyer_tk.TASK_ORD_CD",
+    "task_orders": "select atd_tk.TASK_ORDER_DEPT, atd_tk.TASK_ORDER_ID, atd_tk.TASK_ORDER_DESC, atd_tk.TASK_ORDER_STATUS, atd_tk.TASK_ORDER_TYPE, atd_tk.TK_CURR_AMOUNT, atd_tk.CHARGED_AMOUNT, atd_tk.TASK_ORDER_BAL, buyer_tk.BYR_FDU FROM DEPT_2400_TK_VW atd_tk LEFT JOIN( SELECT DISTINCT TASK_ORD_CD, BYR_FDU FROM REL_BUYER_SELLER_FDU_TK) buyer_tk ON atd_tk.TASK_ORDER_ID = buyer_tk.TASK_ORD_CD where TASK_ORDER_STATUS is not null",
```

Comment:
here's the important change—this ensures that the crufty task orders that are missing `TASK_ORDER_STATUS` are excluded entirely from the pipeline. at last glance there were 10 of them (out of +10k task orders).

Reply:
Would we want to log the number of null entries if we need to go back and check? If this is the query that takes a long time to run then maybe don't.

Reply:
i'm comfortable ignoring them, since we are logging the number of records we're actually handling each time. but i could add a note to the script with the background on why we're excluding those TKs

Reply:
well, actually, this PR can serve as the docs?